src/autogluon/cloud/default_cluster_configs/ray_aws_default_cluster_config.yaml

```diff
@@ -4,6 +4,11 @@ cluster_name: ag_ray_aws_default
 # The maximum number of workers nodes to launch in addition to the head
 max_workers: 1
 
+docker:
+    # The image uri will be dynamically replaced by cloud predictor
+    image: "763104351884.dkr.ecr.us-east-1.amazonaws.com/autogluon-training:0.7.0-cpu-py39-ubuntu20.04"
```

Comment:
Shall we have a different default for GPU instance?

Reply:
This is just a place holder. We'll replace it with GPU one if needed.